ansible/roles/graph-protocol/files/values-prd.yaml

```diff
@@ -0,0 +1,12 @@
+# ethereum network
+network: xdai
+rpcNode: xdai-nethermind.xdai
+# subgraph values
+subgraph_name: circlesubi/circles-ubi
+subgraph_network: xdai
+proxy_factory_address: "0x8b4404DE0CaECE4b966a9959f134f0eFDa636156"
```

Comment:
Now we need to update the variables, right? Like in https://github.com/CirclesUBI/circles-subgraph/blob/main/.env.example

We need to add the value of `proxy_factory_address_crc`, and change the value of this `proxy_factory_address`.